resource-service/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM amazoncorretto:17
-ADD web/build/libs/web-0.0.1-SNAPSHOT.jar app.jar
-ENTRYPOINT ["java","-jar","app.jar"]
+FROM amazoncorretto:17-alpine-jdk
+ADD web/build/libs/web-0.0.1-SNAPSHOT.jar resource-service.jar
```

Comment:
It is a good practice to add `EXPOSE <port>` because it will help anyone who uses your dockerfile to define valid port mapping